### ISSUE TITLE
Test on Windows

### ISSUE THREAD

**Issue by hinerm** [fiji/dockerfiles#2]
Haven't tried any of these dockerfiles on Windows yet, so I'm not sure what's required to get GUI apps running.

**Comment by dscho** [comment]
First of all, there is no official Docker for Windows support yet (only through VirtualBox, which does not really count). And then you probably need a X11 server to run GUI apps inside Docker, because Docker containers really are Linux containers, so the GUI is still X11 (or Wayland, but that is even more complicated to set up).

**Comment by hinerm** [comment]
Indeed.. since the [windows instructions use boot2docker](https://docs.docker.com/installation/windows/) I expect the steps for running Fiji will be similar to those [on OSX](http://imagej.net/Docker#On_OSX).

Looks like we can start by trying [Xming](http://sourceforge.net/projects/xming/) and a windows [socat equivalent](http://serverfault.com/questions/490477/windows-linux-socat-to-share-a-serial-device-over-ip)

**Comment by dscho** [comment]
Xming is an option, but keep in mind that I never managed to compile that beast. Its latest release is pretty old by X11's standards.

VNC might be a better choice (I had [looked into using Xpra](https://github.com/dscho/docker-desktop) instead, but due to the number of regressions introduced in to Xpra I had to let that one go).

You do not need socat. With `boot2docker`, there are [better ways to open ports to services running inside the container](https://docs.docker.com/installation/mac/#access-container-ports).
